vector/src/main/java/im/vector/riotx/features/home/room/detail/timeline/helper/MessageInformationDataFactory.kt

```diff
@@ -72,6 +75,52 @@ class MessageInformationDataFactory @Inject constructor(private val session: Ses
             textColor = colorProvider.getColor(getColorFromUserId(event.root.senderId))
         }
 
+        val room = event.root.roomId?.let { session.getRoom(it) }
+        val e2eDecoration: E2EDecoration
+        val isUserVerified = session.cryptoService().crossSigningService().getUserCrossSigningKeys(event.root.senderId ?: "")?.isTrusted() == true
```

Comment:
This is done even if the room is not encrypted, so maybe move this after a first if to check that?

Reply:
fun i did that because i though you wouldn't like the long && if
fixed